charts/headlamp/values.schema.json

```
@@ -296,6 +296,22 @@
         "port": {
           "type": "integer",
           "description": "Kubernetes Service port"
+        },
+        "clusterIP": {
+          "type": "string",
+          "description": "Kubernetes Service clusterIP"
+        },
+        "loadBalancerIP": {
+          "type": "string",
+          "description": "Kubernetes Service loadBalancerIP"
+        },
+        "loadBalancerSourceRanges": {
+          "type": "string",
+          "description": "Kubernetes Service loadBalancerSourceRanges"
+        },
+        "nodePort": {
+          "type": "integer",
```

Sometimes nodePort would not be required, when we run `helm template` it fails. So it would be better to have null as well.

```suggestion
"type": ["integer", "null"],
```